bot/cogs/watchchannels/watchchannel.py

```diff
@@ -0,0 +1,359 @@
+        except discord.HTTPException as exc:
+            self.log.exception(
+                f"Failed to send message to {self.__class__.__name__} webhook",
+                exc_info=exc
+            )
+
+    async def relay_message(self, msg: Message) -> None:
+        """Relays the message to the relevant WatchChannel"""
+        last_author, last_channel, count = self.message_history
+        limit = BigBrotherConfig.header_message_limit
+
+        if msg.author.id != last_author or msg.channel.id != last_channel or count >= limit:
+            self.message_history = [msg.author.id, msg.channel.id, 0]
+
+            await self.send_header(msg)
+
+        cleaned_content = msg.clean_content
+
+        if cleaned_content:
+            # Put all non-media URLs in a codeblock to prevent embeds
+            media_urls = {embed.url for embed in msg.embeds if embed.type in ("image", "video")}
+            for url in URL_RE.findall(cleaned_content):
+                if url not in media_urls:
+                    cleaned_content = cleaned_content.replace(url, f"`{url}`")
+            await self.webhook_send(
+                cleaned_content,
+                username=msg.author.display_name,
+                avatar_url=msg.author.avatar_url
+            )
+
+        if msg.attachments:
+            try:
+                await messages.send_attachments(msg, self.webhook)
+            except (errors.Forbidden, errors.NotFound):
+                e = Embed(
+                    description=":x: **This message contained an attachment, but it could not be retrieved**",
+                    color=Color.red()
+                )
+                await self.webhook_send(
+                    embed=e,
+                    username=msg.author.display_name,
+                    avatar_url=msg.author.avatar_url
+                )
+            except discord.HTTPException as exc:
+                self.log.exception(
+                    f"Failed to send an attachment to {self.__class__.__name__} webhook",
+                    exc_info=exc
+                )
+
+        self.message_history[2] += 1
+
+    async def send_header(self, msg) -> None:
+        """Sends a header embed to the WatchChannel"""
+        user_id = msg.author.id
+
+        guild = self.bot.get_guild(GuildConfig.id)
+        actor = guild.get_member(self.watched_users[user_id]['actor'])
+        actor = actor.display_name if actor else self.watched_users[user_id]['actor']
+
+        inserted_at = self.watched_users[user_id]['inserted_at']
+        time_delta = self._get_time_delta(inserted_at)
+
+        reason = self.watched_users[user_id]['reason']
+
+        embed = Embed(description=(
+            f"{msg.author.mention} in [#{msg.channel.name}]({msg.jump_url})\n"
+        ))
+        embed.set_footer(text=(
+            f"Added {time_delta} by {actor} | "
+            f"Reason: {reason}"
```

Comment:
This could fit on one line and it's a bit misleading seeing it broken up as I thought there was a newline in there at first.

Reply:
Changed and I've changed a couple of similar lines as well.